app/assets/stylesheets/views/_landing_page/hero.scss

```diff
@@ -34,3 +40,45 @@
     margin-top: -(govuk-spacing(6));
   }
 }
+
+.app-b-hero--mid-page {
+  overflow: hidden;
+  min-height: $desktop-height;
+
+  @include govuk-media-query($until: desktop) {
+    display: flex;
+    flex-direction: column-reverse;
+    min-height: $tablet-height;
+  }
+
+  @include govuk-media-query($until: tablet) {
+    min-height: $mobile-height;
+  }
+
+  .app-b-hero__image {
+    position: absolute;
+    left: 50%;
+    transform: translate(-50%);
+  }
+
+  .app-b-hero__textbox-wrapper {
```

Comment:
Whilst this is a nice solution, I'm afraid it reintroduces the original problem of overflow if the text content is too big. Sorry.

![Screenshot 2024-10-17 at 12 54 43](https://github.com/user-attachments/assets/cd71a724-2388-493a-ba86-38322892b1e7)

My only other thought on how to do this would be if we calculated what the height would be, and set that as the min-height. So if the overall hero is 610px, the min-height would need to be 610px - top margin - bottom margin (of the textbox)?

Reply:
Thanks. I've added these changes as a separate [commit](https://github.com/alphagov/frontend/pull/4286/commits/1c9242b51bd23f37c9180a3acff12d6644c5560f) changing heights to minimum heights which seems to resolve it.